### ISSUE TITLE
One-to-one property of logs in EL3, EL5, and EL6

### ISSUE THREAD

**Issue by AbbyANoble** [TeamBasedInquiryLearning/library#430]
We define this property in [Definition 5.6.7](https://library.tbil.org/2024/precalculus/EL6.html#def-one-to-one-property), and I think it would help to write it out in the notation instead of just words: 

$log_b M = log_b N$ if and only if $M=N$.

Also, I would suggest changing the second half of the definition into an Observation. Something like "Notice this definition allows us to set the arguments (M and N) equal to each other once we get each side expressed as a single logarithm with matching bases."

Also, this seems to be the first time it's defined (in EL6), but it is mentioned in the previous section (EL5) as if it was defined in EL3. See

- [Remark 5.5.1](https://library.tbil.org/2024/precalculus/EL5.html#EL5-3-3) 
- [Activity 5.5.2(b)](https://library.tbil.org/2024/precalculus/EL5.html#investigate-product-property-of-logs-3) 
- [Activity 5.5.3(b)](https://library.tbil.org/2024/precalculus/EL5.html#investigate-quotient-property-of-logs-3) 

We need to clear this up and figure out where we want everything located and referenced.

**Comment by siwelwerd** [comment]
I agree that is not a definition.  And while I know it's common, I'd rather we stay away from the language "We can set them equal to each other" instead of saying precisely what we are doing, namely exponentiating both sides of the equation and simplifying.

>We need to clear this up and figure out where we want everything located and referenced.

Maybe it fits nicely right after [Remark 5.3.9](https://library.tbil.org/2024/precalculus/EL3.html#EL3-3-11) ?  Then in [Activity 5.6.8](https://library.tbil.org/2024/precalculus/EL6.html#solving-using-one-to-one-property) we can still call back to it.

**Comment by StevenClontz** [comment]
> This isn't a new thing, it's just the idea we use all the time in solving equations that if a=b then f(a)=f(b) for whatever function f you like. Here f is exponentiation.

I think an explicit reminder of this (or even having this as a Fact somewhere to reference, maybe in https://library.tbil.org/2024/precalculus/instructor/FN2.html ?) can't hurt, at least within the scaffolding of the first activity where they need to apply this fact.

**Comment by AbbyANoble** [comment]
Yeah. I don't think the new application (exponentiating) is something that they will recognize as following from other situations. It feels new at this level in the context of new functions. It should be mentioned.

**Comment by tdegeorge** [comment]
What about this? I moved some things around between EL5 and EL6.....
